osu.Game/Rulesets/Objects/Drawables/DrawableHitObject.cs

```diff
@@ -96,10 +100,10 @@ public abstract class DrawableHitObject : SkinReloadableDrawable
         /// </remarks>
         protected virtual float SamplePlaybackPosition => 0.5f;
 
-        private BindableList<HitSampleInfo> samplesBindable;
-        private Bindable<double> startTimeBindable;
-        private Bindable<bool> userPositionalHitSounds;
-        private Bindable<int> comboIndexBindable;
+        public readonly Bindable<double> StartTimeBindable = new Bindable<double>();
```

Comment:
This `public` doesn't have an explanation for now but I can see further changes will be using this field, so I'm purposefully ignoring this.

Reply:
Ah, this was due to a change I had in this PR (refactor of HOC) that was temporarily reverted. I'll change back to private.

In the full hitobject-pooling implementation I had all of these public just to conform to, e.g. all `DrawableOsuHitObject` ones being public (and required to be public).